--- bochs-2.2.6-build.sh	2012-09-26 17:38:30.000000000 -0700
+++ bochs-2.2.6-build-patched.sh	2020-08-23 16:58:05.561336782 -0700
@@ -14,18 +14,18 @@
 mkdir bochs-2.2.6
 tar xzf $SRCDIR/bochs-2.2.6.tar.gz
 cd bochs-2.2.6
-cat $PINTOSDIR/src/misc/bochs-2.2.6-ms-extensions.patch | patch -p1
-cat $PINTOSDIR/src/misc/bochs-2.2.6-big-endian.patch | patch -p1
-cat $PINTOSDIR/src/misc/bochs-2.2.6-jitter.patch | patch -p1
-cat $PINTOSDIR/src/misc/bochs-2.2.6-triple-fault.patch | patch -p1
-cat $PINTOSDIR/src/misc/bochs-2.2.6-solaris-tty.patch | patch -p1
-cat $PINTOSDIR/src/misc/bochs-2.2.6-page-fault-segv.patch | patch -p1
-cat $PINTOSDIR/src/misc/bochs-2.2.6-paranoia.patch | patch -p1
-cat $PINTOSDIR/src/misc/bochs-2.2.6-gdbstub-ENN.patch | patch -p1
-if test "`uname -s`" = "SunOS"; then
-    cat $PINTOSDIR/src/misc/bochs-2.2.6-solaris-link.patch | patch -p1
-fi
-CFGOPTS="--with-x --with-x11 --with-term --with-nogui --prefix=$DSTDIR"
+# cat $PINTOSDIR/src/misc/bochs-2.2.6-ms-extensions.patch | patch -p1
+# cat $PINTOSDIR/src/misc/bochs-2.2.6-big-endian.patch | patch -p1
+# cat $PINTOSDIR/src/misc/bochs-2.2.6-jitter.patch | patch -p1
+# cat $PINTOSDIR/src/misc/bochs-2.2.6-triple-fault.patch | patch -p1
+# cat $PINTOSDIR/src/misc/bochs-2.2.6-solaris-tty.patch | patch -p1
+# cat $PINTOSDIR/src/misc/bochs-2.2.6-page-fault-segv.patch | patch -p1
+# cat $PINTOSDIR/src/misc/bochs-2.2.6-paranoia.patch | patch -p1
+# cat $PINTOSDIR/src/misc/bochs-2.2.6-gdbstub-ENN.patch | patch -p1
+# if test "`uname -s`" = "SunOS"; then
+#    cat $PINTOSDIR/src/misc/bochs-2.2.6-solaris-link.patch | patch -p1
+# fi
+CFGOPTS="--enable-cpu-level=6 --with-x --with-x11 --with-term --with-nogui --prefix=$DSTDIR"
 mkdir plain &&
         cd plain && 
         ../configure $CFGOPTS --enable-gdb-stub && 
